lib/symengine/matrix_base.rb

```diff
@@ -0,0 +1,7 @@
+module SymEngine
+  class MatrixBase
+    def inspect
+      "#<#{self.class}()>"
```

Comment:
I assume both dense and sparse matrices should have the same inspect ` "#<#{self.class}(#{rows}x#{cols})>"`, so you can define it here in `MatrixBase` only. Or am I missing something?